client/src/Movies/MovieList.js

```diff
@@ -14,13 +14,14 @@ const MovieList = props => {
           console.error('Server Error', error);
         });
     }
-    
+
     getMovies();
   }, []);
-  
+
   return (
     <div className="movie-list">
-      {this.state.movies.map(movie => (
+      {state.movies.map(movie => (
```

Comment:
I didn't merge because this should be `{movies.map(movie =>`

I think dustin and some other folks got it updated within the last hour.